.changes/unreleased/Dependency-20220815-182731.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Dependency
+body: "Bump mashumaro[msgpack] from 3.0.3 to 3.0.4 in /core"
+time: 2022-08-15T18:27:31.00000Z
+custom:
+  Author: dependabot[bot]
+  Issue: 4904
+  PR: 5649
+
```

Comment:
@emmyoop this extra newline causes the code check to fail for formatting. I'm just going to manually fix it but I'm curious if there is something we can tweak here to make it right on generation